qiita_pet/handlers/study_handlers/artifact.py

```diff
@@ -34,7 +34,8 @@ class NewArtifactHandler(BaseHandler):
     def get(self):
         study_id = self.get_argument("study_id")
         prep_id = self.get_argument("prep_template_id")
-        artifact_types = artifact_types_get_req()['types']
+        artifact_types = [(at, desc) for at, desc, _, _, raw in
```

Comment:
is "raw" short for "is_user_uploadable", if so you might want to use it here explicitly to emphasize the filtering criterion